big-red-shuttle/StopsViewController.swift

```diff
@@ -61,7 +64,33 @@ class StopsViewController: UIViewController, GMSMapViewDelegate {
             marker.map = mapView
         }
     }
+    
+    
+    //drawing
+    func initPolyline() {
+        let stops = getStops()
+        let stopsA = Array(stops[1..<9]) //can only have a max of 8 waypoints (i.e. stops that don't include start and end)
```

Comment:
ok in that case, make `9` a constant like `maxWaypoints` and use that constant instead